cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/instantsearch-config.js

```diff
@@ -457,6 +439,10 @@ function enableInstantSearch(config) {
 
                                 item.displayPrice = variantPrice;
                                 item.calloutMsg = variantCalloutMsg;
+
+                                item.highestPriceBookPrice = selectedVariant.pricebooks && selectedVariant.pricebooks[algoliaData.currencyCode] && selectedVariant.pricebooks[algoliaData.currencyCode].length > 0 ? selectedVariant.pricebooks[algoliaData.currencyCode].reduce((max, pricebook) => {
```

Comment:
This doesn't makes things more clear: a 262 characters line with a ternary check at the end that triggers a function is quite hard to process.
You should break it down.